apps/tailwind-components/components/input/RadioIcon.vue

```diff
@@ -10,13 +10,17 @@
       cx="10"
       cy="10"
       r="9"
-      stroke-width="1"
+      :style="{ 'stroke-width': invalid ? 2 : 1 }"
       fill="none"
-      class="stroke-current"
       :class="{
         'fill-input hover:fill-input-checked hover:stroke-none focus:fill-input-checked focus:stroke-none':
           !checked,
-        'fill-input-checked stroke-none': checked,
+        'fill-input-checked stroke-none': checked && !invalid && !valid,
+        'fill-invalid': checked && invalid,
+        'fill-valid': checked && valid,
+        'stroke-current ': valid,
```

Comment:
I think this is supposed to be `default` or not yet validated.

I think if we remove the validation from the icon and place it at the group level, it will simplify this component and eliminate the need to define styles for input icons.